sdk/python/lib/pulumi/config.py

```diff
@@ -169,7 +169,7 @@ def get_secret_float(self, key: str) -> Optional[Output[float]]:
         :rtype: Optional[float]
         :raises ConfigTypeError: The configuration value existed but couldn't be coerced to float.
         """
-        v = self.get(key)
+        v = self.get_float(key)
```

Comment:
This one is a real bug fix - static types FTW :smile:.

Reply:
nice!  it's also a test hole.  but this def reduces the impact of having to have tests to validate behaviors that a type checker can find for us :)